index.js

```diff
@@ -1,52 +1,111 @@
+//---------------------------------------------------------Question 1 ----------------------------------------------------------------------------------------------------------------
+/* Is time to create things in the DOM!!. We want you to create a new 'div' Element and append it as a child of the element that has the 'id = a-1' 
+(the element is in index.html file) all right our new element needs style. For that, assign to the new element the className 'square'!. 
+Each time the user clicks on the new element it has to respond changing the displayed text 'x' to 'o' 
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------*/
+
+//Assign 'question1' element to  a 'div' varialbe, Assign 'square' element to a 'square' variable.
+let div = document.getElementById('question1');
+//document.getElementsByClassName('square')[0];
+
+// call listen function
+//create new div element 'nDive', add all square syling to the new element
+//Assign string value to the new div, then replace the old div element with the new one!!
+
+div.addEventListener ('click', () => {
+    nDiv = document.getElementsByClassName('square')[0];
+    nDiv.innerHTML = "o";         
+    });
+
+
+    /* -----------Create new Div with different text -------------
+    let div = document.getElementById('question1');
+    let square = document.getElementsByClassName('square')[0];
+    div.addEventListener ('click', () => {
+    
+        let nDiv = document.createElement('div');
+        nDiv.classList.add('square')
+        nDiv.innerHTML = "o";         
+        div.replaceChild(nDiv, square); 
+    }); 
+    --------------------------------------------------------------*/
+
+/*----------------------------------------Question 2 ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+Write a JavaScript program to remove items from a dropdown list. In order to create the list we are providing you with the array below, those items are the ones you need to add to the dropdown list as the select options
+you also need to create a button that makes the remove action of the element selected append the new list as a child of the element with the 'id = a-2' append the button as a child of the element with the 'id = a-2' 
 const colors = [ 'red' , 'white', 'black' , 'green' , 'orange'];
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- */
 
+// assign the button element to 'removeColorButton', and call 'click function it to  assign "removeColorButton".
+const removeColorButton = document.getElementsByTagName('button')[0];
```

Comment:
I think they want you to create the select and options using javascript not jjust plain html